src/tools/gcc.lua

```diff
@@ -675,8 +675,14 @@
 	}
 
 	function gcc.gettoolname(cfg, tool)
```

Comment:
Not sure if `windres` should have suffix too.

Reply:
Good question. I'd lean on the side of "if the user specified a toolset with a version, apply version to all tools in the toolset"